src/index.ts

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+import { PreCompiler, ScenarioOutline, Tag } from "gherking";
+import { ScenarioOutlineExpanderConfig } from "./types";
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const debug = require("debug")("gpc:scenario-outline-expander");
+
+const DEFAULT_CONFIG: ScenarioOutlineExpanderConfig = {
+    ignoreTag: '@notExpand'
```

Comment:
Following the practice we used in other precompilers, I'd suggest using only the name, without the `@`